### DISCUSSION TITLE
Declustering Method

### DISCUSSION THREAD

**Discussion by CamPhillips26** [georgebv/pyextremes#34] [General]
Is a 'block' or 'runs' method used to decluster exceedances?

**Comment by georgebv** [comment]
Please provide more context. I.e., what do you mean by `block` and `runs`.

You can find current implementation here: https://github.com/georgebv/pyextremes/blob/master/src/pyextremes/extremes/peaks_over_threshold.py

Documentation: https://georgebv.github.io/pyextremes/user-guide/4-peaks-over-threshold/
